hooks/src/index.js

```diff
@@ -307,6 +310,16 @@ export function useErrorBoundary(cb) {
 	];
 }
 
+
+export function useId() {
+	const state = getHookState(currentIndex++, 11);
+	if (!state._id) {
+		currentIdCounter++;
+		state._id = (currentInternal._rootId + currentInternal._domDepth + currentIdCounter).toString(32)
```

Comment:
Do you know how collision resistant this is?

Also, do we need to worry about preact users using a simple ID (single digit) themselves?

Reply:
Prefixed  with `_P` now